rpki-validator/src/main/java/net/ripe/rpki/validator3/domain/validation/CertificateTreeValidationService.java

```diff
@@ -337,9 +337,9 @@ private void validateCertificateAuthority(final TrustAnchor trustAnchor,
             }
             accumulator.add(manifestObject.get().key(), manifest, ImmutableSortedSet.of(temporary.getCurrentLocation().getName()));
 
-            manifest.getFiles().entrySet()
-                .parallelStream()
+            manifest.getFiles().entrySet().stream()
                 .flatMap(entry -> getManifestEntry(manifestUri, entry))
+                .parallel()
```

Comment:
Why this change? Not arguing, but just out of curiosity.

Reply:
From the commit message:

```
  Load manifest entries sequentially but validate concurrently

Loading manifest entries is mostly IO bound (Xodus read transactions)
so there is no need for parallel processing. Actual validation is
still in parallel since it is CPU intensive.
```

Still, now that `blocking` is removed from the Xodus `readTx` it probably doesn't matter much anyway.